src/iscn/likerIdsAddresses.ts

```diff
@@ -0,0 +1,27 @@
+import axios, { AxiosResponse } from 'axios';
+
+export async function getLikeWalletByLikerId(
+  likerId: string,
+  { LIKE_CO_API_ROOT = 'https://api.like.co' }: { LIKE_CO_API_ROOT?: string } = { LIKE_CO_API_ROOT: 'https://api.like.co' },
+)
+: Promise < string | void | null > {
+  try {
+    return await axios.get(`${LIKE_CO_API_ROOT}/users/id/${likerId}/min`)
+      .then((element: AxiosResponse): string | undefined => element?.data?.likeWallet);
+  } catch (error) {
+    return null;
```

Comment:
Always log the error instead of sliently dropping all
Consider adding a special case for status code = `404` and skip logging if you think it is expected